### DISCUSSION TITLE
近期 roadmap

### DISCUSSION THREAD

**Discussion by luxcgo** [go-olive/olive#74] [Ideas]
1. toml 文档刷新
    * 生效中的配置   把配置删掉   录制中的任务继续录制 直到下播 （下次开播不会录制  配置已经无了）
    * 新增的配置是肯定需要处理的
2. streamlink传递录制
3. 解决  虎牙星秀区 画质切换  花屏问题

**Comment by UVJkiNTQ** [comment]
FR：
录制启动与否的关键字（白名单/黑名单）：仅录制含有'字符串A' or / and '字符串B'的直播；仅录制不含有 '字符串A' and '字符串B'的直播

用途：比如多平台主播，部分平台有限定。

**Comment by jkfujr** [comment]
#76
